packages/core/src/App/Containers/Layout/header/traders-hub-onboarding.tsx

```diff
@@ -2,26 +2,16 @@ import React from 'react';
 import { Icon, Popover } from '@deriv/components';
 import { observer, useStore } from '@deriv/stores';
 import { Localize } from '@deriv/translations';
-import { useFeatureFlags } from '@deriv/hooks';
 import { useLocalStorage } from 'usehooks-ts';
 
 const TradersHubOnboarding = observer(() => {
-    const { traders_hub, ui } = useStore();
-    const { setIsFirstTimeVisit, toggleIsTourOpen, is_tour_open } = traders_hub;
+    const { ui } = useStore();
     const { is_dark_mode_on, is_mobile } = ui;
-    const { is_next_wallet_enabled } = useFeatureFlags();
     const [, setWalletsOnboarding] = useLocalStorage('walletsOnboarding', '');
 
-    const onClickHandler = is_next_wallet_enabled
-        ? () => {
-              setWalletsOnboarding('started');
-          }
-        : () => {
-              if (!is_tour_open) {
-                  toggleIsTourOpen(true);
-              }
-              setIsFirstTimeVisit(false);
-          };
+    const onClickHandler = () => {
```

Comment:
removing the `is_next_wallet_enabled` flag is fine from here?

Reply:
This component is used only by wallet so I removed the condition here , thanks for checking 🙏